rotkehlchen/tests/unit/test_protocol_balances.py

```diff
@@ -342,3 +343,29 @@ def test_gmx_balances(
             usd_value=FVal('42.939657170821103766867050208907'),
         ),
     }
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0xecdED8b1c603cF21299835f1DFBE37f10F2a29Af']])
+def test_aave_v3_balances(
+        ethereum_inquirer: 'EthereumInquirer',
+        ethereum_transaction_decoder: 'EthereumTransactionDecoder',
+        ethereum_accounts: list[ChecksumEvmAddress],
+        inquirer: 'Inquirer',  # pylint: disable=unused-argument
+) -> None:
+    """Check that Aave v3 positions are properly detected"""
+    get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_inquirer,
+        database=ethereum_transaction_decoder.database,
+        tx_hash=deserialize_evm_tx_hash('0xf700ba5c098e04b05767a4852dbd56fa560bbe8f8adcc595dd2f6b65c68dfcc8'),
+    )
+    user_balance = Aavev3Balances(
+        database=ethereum_transaction_decoder.database,
+        evm_inquirer=ethereum_inquirer,
+    ).query_balances()[ethereum_accounts[0]]
+
+    expected_balance = FVal('170.750023830650525005')
+    assert user_balance[EvmToken('eip155:1/erc20:0x4d5F47FA6A74757f35C14fD3a6Ef8E3C9BC514E8')] == Balance(  # noqa: E501
+        amount=expected_balance,
+        usd_value=expected_balance * FVal('1.5'),
```

Comment:
could you use the mocked prices fixture to add the 1.5 there? it is easier to read later. Also can you add another user (so two users) that has more than 1 position? that would ensure that your logic works as expected